extensions/opc/src/fetchopc.cpp

```diff
@@ -62,7 +62,7 @@ namespace org::apache::nifi::minifi::processors {
 
     if (idType_ == opc::OPCNodeIDType::Int) {
       try {
-        std::stoi(nodeID_);
+        static_cast<void>(std::stoi(nodeID_));
```

Comment:
```suggestion
        // ensure that nodeID_ can be parsed as an int
        static_cast<void>(std::stoi(nodeID_));
```

Reply:
Updated in be9cace2da1aa161129ac9c22986033dafc199db